### PR TITLE
Use tan stack virtualizer with TagSelector

### PR DESCRIPTION
**Pull request by salazarm**
## Summary & Motivation

I used a virtualization approach that we stopped using (potentially due to the cross browser issues?).
Updating TagSelector to use the new updated approach.

## How I Tested These Changes
Starting up my windows PC to checkout this PR and see if it fixes the issue.

### CODE REVIEW

File: js_modules/dagit/packages/ui/src/components/VirtualizedTable.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import {Box, Colors} from '@dagster-io/ui';
```
Comment (salazarm) [inline js_modules/dagit/packages/ui/src/components/VirtualizedTable.tsx:1]: copied this from `core`.

Comment (bengotow) [reply js_modules/dagit/packages/ui/src/components/VirtualizedTable.tsx:1]: Hmm can we remove this file from core now? I think having two copies that export the same things could get confusing

File: js_modules/dagit/packages/ui/src/components/TagSelector.tsx [hunk 2]
```diff
@@ -130,11 +145,12 @@ export const TagSelector = ({
   }, [
     allTags,
     dropdownStyles,
+    items,
     renderDropdown,
     renderDropdownItem,
-    rowHeight,
     selectedTags,
     setSelectedTags,
+    totalHeight,
```
Comment (bengotow) [inline js_modules/dagit/packages/ui/src/components/TagSelector.tsx:153]: This looks good to me - I think @hellendag has the context on why `react-virtualized` vs `@tanstack/react-virtual`. I wonder if an alternative to the bugs we've been seeing is upgrading to a newer version of react-virtualized (maybe?)

**Review by bengotow — APPROVED** [review]

File: js_modules/dagit/packages/ui/src/components/VirtualizedTable.tsx (hunk 1 above)
Comment (salazarm) [reply js_modules/dagit/packages/ui/src/components/VirtualizedTable.tsx:1]: I'll do this in a follow up

Comment (salazarm) [reply js_modules/dagit/packages/ui/src/components/VirtualizedTable.tsx:1]: (I have to update a bunch of imports)

**Pull request merged**